protocols/JabberG/res/jabber.rc

```diff
@@ -39,6 +39,7 @@ BEGIN
     PUSHBUTTON      "Cancel",IDCANCEL,223,207,50,14
 END
 
+// GUI in Main Menu, Accounts...

```

Comment:
It's not useful to put a comment there, as Visual studio overrides it with auto-generated resource.

Reply:
What is right way to add comment?

Reply:
Just don't add one. I never saw one in the entire code.

Reply:
@tweimer you comment looks like a personal opinion.

Reply:
@tweimer you comment looks like a personal opinion.

Reply:
No, it's not. Do you see any other resource file in this repository with hand written comments?

Reply:
@tweimer I asked you about argues, but you continue to ask questions to me. You joined to code review (https://en.wikipedia.org/wiki/Code_review). 

You hold 10 strings about 2 weeks without argues about code. We did not improve Miranda, we did not know new about code. We did nothing (excluding @dartraiden who post helpful comments)

Reply:
I did give you an argument in the very first comment about this. It's not useful to put a comment there, as Visual studio overrides it with auto-generated resource. Also, I did give you some helpful comments on how to improve your pull request.

If you prefer to argue instead of fixing those points, I will close here.